### PR TITLE
Dev-142: remove old majors and change the names of new majors

### PR DESCRIPTION
**Pull request by RosaGao**

### CODE REVIEW

File: lib/resources/majors.tsx [hunk 1]
```diff
@@ -3372,257 +3598,31 @@ const bsCS_Old: Major = {
   ],
 };
 
-// https://www.cs.jhu.edu/2021undergraduate-advising-manual/
-const bsCS_New: Major = {
-  degree_name: 'B.S. Computer Science (NEW - 2021 & after)',
-  abbrev: 'B.S. CS',
+// https://www.cs.jhu.edu/undergraduate-studies/academics/ugrad-advising-manual/
+const baCS_New: Major = {
```
Comment (mpark63) [inline lib/resources/majors.tsx:3602]: feel free to just name the var baCS

File: lib/resources/majors.tsx [hunk 2]
```diff
@@ -3121,56 +3401,26 @@ const bsCS_Old: Major = {
...
-            '<b>Intro Algorithms</b> <br /> EN.601.433 Intro Algorithms',
-          required_credits: 3,
-          criteria: 'EN.601.433[C]',
+            '<b>Lower Level Undergraduate:</b><p>500.112/113/114 Gateway Computing or AP Comp Sci A or ' +
+            'equivalent<p>601.220 Intermediate Programming</p><p>601.226 Data Structures</p><p>601.229 ' +
+            'Computer System Fundamentals</p><p>601.230 Mathematical Foundations for Computer Science, or 601.231 Automata for those grandfathered into the old major</p><p>601.433 Algorithms</p>',
+          required_credits: 21,
+          criteria:
+            'EN.500.112[C]^OR^EN.500.113[C]^OR^EN.500.114[C]^OR^EN.601.220[C]^OR^EN.601.226[C]' +
+            '^OR^EN.601.229[C]^OR^EN.601.230[C]^OR^EN.601.433[C]^OR^EN.601.231[C]',
```
Comment (mpark63) [inline lib/resources/majors.tsx:3417]: awesome awesome

**Review by mpark63 — CHANGES_REQUESTED** [review]
ok looks good to me, did you try testing yet? 

if not, please run this script with dev database and test the DEV-142 branches locally. Nothing should break hopefully.

Also please summarize changes you made to majors.tsx!

**Review by josephined21 — APPROVED** [review]
looks good! :D

**Review by mpark63 — APPROVED** [review]
lgtm!

**Review by josephined21 — APPROVED** [review]
pog!

**Pull request merged**